tensorflow/core/framework/register_types.h

```diff
@@ -58,7 +58,11 @@ limitations under the License.
 // Note: macros are defined in same order as types in types.proto, for
 // readability.
 #define TF_CALL_float(m) m(float)
+#if defined(_WINDOWS) && !defined(GOOGLE_CUDA) && !defined(__AVX__) && !defined(__AVX2__) // Not supported on Eigen for SSE
```

Comment:
Hmm... I guess you're right.  It is a little extreme.

I'll look for a more precise way of doing this.